build.bash

```diff
@@ -100,7 +111,7 @@ check_command_availability_and_exit() {
 
 # mount rpi image using userspace tools, in docker use privileged mount via kpartx
 mount_image_file_boot() { # imagefile buildfolder
-  if ! running_in_docker && ! running_on_github && ! is_pi; then
+  if ! running_in_docker && ! running_on_github || is_pi; then
```

Comment:
&& !   is not the same as ||,   why changing the logic here?

Reply:
typo, changed